src/components/Problems/Problems.tsx

```diff
@@ -198,14 +199,18 @@ export const Problems: FC = () => {
           />
         ))}
       </Stack>
-      {displaySideContent.type === 'discussion' && (
+      <Dialog
+        open={displayDiscussion.type === 'discussion'}
+        close={() => setDisplayDiscussion({type: '', problemId: -1, problemNumber: -1})}
+        title={`Diskusia - úloha ${displayDiscussion.problemNumber}`}
+      >
         <Discussion
-          problemId={displaySideContent.problemId}
-          problemNumber={displaySideContent.problemNumber}
-          closeDiscussion={() => setDisplaySideContent({type: '', problemId: -1, problemNumber: -1})}
+          problemId={displayDiscussion.problemId}
+          problemNumber={displayDiscussion.problemNumber}
+          closeDiscussion={() => setDisplayDiscussion({type: '', problemId: -1, problemNumber: -1})}
```

Comment:
`() => setDisplayDiscussion({type: '', problemId: -1, problemNumber: -1})` by som extrahoval, kedze je to pouzite dvakrat